### PR TITLE
ALL: Unify messages concerning engine data files

### PR DESCRIPTION
**Pull request by BenCastricum**
The helps translators a bit by removing filenames from messages so no duplicate messages have to be translated. Some engine didn't display a dialog box about the problem found. Some message were not yet translated.
Feedback appreciated.

### CODE REVIEW

**Comment by wjp** [comment]
Unifying this makes a lot of sense, thanks. But if we're going to pick one, this might not be the best phrase. I think the data files should always be in the installation package, tarball, or git repository, so it might be more likely that is a path problem than that a file needs to be downloaded.

**Comment by BenCastricum** [comment]
I added a commit for alternate messages referring to the README instead of the website. Makes a bit more sens I think. Not quite sure if the README is the right place for it though.

**Comment by rootfather** [comment]
I think a README bundled with every ScummVM release is a better place for such instructions than a link to the website.

File: README [hunk 1]
```diff
@@ -1742,6 +1743,28 @@ Broken Sword
...
+- (Re)download and install the latest version of ScummVM package
+Because the package should contain the required data, this should fix it.
+
+- Download the engine data file manually from github
+On https://github.com/scummvm/scummvm/blob/master/dists/engine-data/ you can
+find the latest versions of all engine data files.
+
+- Check your path settings
+ScummVM will scan it's own directory and the Extra Path directory for the
+engine data files. Try moving the engine data file into those one of those
```
Comment (raziel-) [inline README]: One too many "those"

**Comment by BenCastricum** [comment]
Fixed that one. But is the README the desired place? The FAQ is also a nice place for this info and already has a trouble shooting section where it could fit in.

**Comment by BenCastricum** [comment]
Here's the proposal for not mentioning any instructions in the message. We don't do that for other messages either, so the user will follow the "normal" procedure for getting help. I would recommend a entry for this issue into the FAQ if it really turns out to be a FAQ.

Before merging (if this is the agreed way to go) I would like to squash the PR first.

**Comment by criezy** [comment]
I think having the instructions in the README is fine.
Then if it turns out that this is a FAQ it can indeed be added there as well. The two are not exclusive.

But overall this is  nice pull request. Unifying those messages is indeed a good idea.

**Pull request merged**